### PR TITLE
Replace EasyDict to SmartDict

### PR DESCRIPTION
**Pull request by iizukak**
Solve #1092 

Odoku-san implemnetd EasyDict. We can replace EasyDict to SmartDict.

### CODE REVIEW

File: blueoil/utils/config.py [hunk 1]
```diff
@@ -133,19 +133,19 @@ def _load_py(config_file):
...
     # use only upper key.
-    return EasyDict({
+    return SmartDict({
         key: value
         for key, value in config.items()
         if key.isupper()
     })
 
 
 def _easy_dict_to_dict(config):
```
Comment (oatawa1) [inline blueoil/utils/config.py:143]: How about changing the function name?
```suggestion
def _smart_dict_to_dict(config):
```

**Review by oatawa1 — COMMENTED** [review]
I leave one comment.

File: blueoil/utils/config.py (hunk 1 above)
Comment (iizukak) [reply blueoil/utils/config.py]: @oatawa1 Thanks. Changed.

**Review by oatawa1 — APPROVED** [review]
LGTM. 

Such a big PR. Thank you.

**Review by tfujiwar — APPROVED** [review]
RA

**Pull request merged**